### DISCUSSION TITLE
Split by Keyframe

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#1989] [Support]
I pressed the button that allowed me to split videos by keyframe, but I don't want the segments to have the same keyframe more than once. How do I fix that?
![Screenshot (581)](https://github.com/mifi/lossless-cut/assets/141977822/73fe63a9-1bbe-455b-845e-24818867ea62)

**Comment by mifi** [comment]
maybe you can shift all segments start times a few frames back/forward

**Reply by Jay123210599** [reply]
how?

**Reply by mifi** [reply]
the segments menu

**Reply by Jay123210599** [reply]
It didn't work.
